framework_tvb/tvb/adapters/creators/stimulus_creator.py

```diff
@@ -104,7 +105,8 @@ def get_input_name():
 
     @staticmethod
     def get_filters():
-        return None
+        return FilterChain(fields=[FilterChain.datatype + '.surface_type'], operations=["=="],
+                                 values=[CORTICAL])
```

Comment:
The filter is ok. In this file there is a TODO related to this filter. The TODO can be removed now, right?